core/string/ustring.cpp

```diff
@@ -3441,18 +3441,27 @@ String String::replacen(const String &p_key, const String &p_with) const {
 String String::repeat(int p_count) const {
 	ERR_FAIL_COND_V_MSG(p_count < 0, "", "Parameter count should be a positive number.");
```

Comment:
One more thing, I missed that you missed this part (it works fine for zero):
```suggestion
	ERR_FAIL_COND_V_MSG(p_count < 0, "", "Parameter count should be a non-negative number.");
```